### PR TITLE
CAPA: Release v29.1.0.

### PR DESCRIPTION
**Pull request by weseven**
### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version (_for 1.29 minor_)

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=v29.0.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

File: capa/v29.1.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    dependsOn:
+    - prometheus-operator-crd
+  - name: vertical-pod-autoscaler-crd
+    version: 3.1.0
+  components:
+  - name: cluster-aws
+    catalog: cluster
+    version: 2.0.0
+  - name: flatcar
+    version: 3975.2.0
```
[the remark this answers is not in the document]
Comment (weseven) [reply capa/v29.1.0/release.yaml:120]: kernel from 6.1 to 6.6 and systemd from 252 to 255 are the main changes, but they shouldn't affect us :crossed_fingers:

**Review by njuettner — APPROVED** [review]

File: capa/v29.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# :zap: Giant Swarm Release v29.1.0 for CAPA :zap:
+
+## Changes compared to v29.0.0
+
+### Components
+
+- Flatcar from v3815.2.5 to [v3975.2.0](https://www.flatcar.org/releases#release-3975.2.0)
+- Kubernetes from v1.29.7 to [v1.29.8](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.29.md#changelog-since-v1297)
+- os-tooling from v1.15.0 to v1.18.1
```
Comment (AndiDog) [inline capa/v29.1.0/README.md:9]: Should we exclude this internal chart detail? Customers can't easily find out what this is.

**Review by AndiDog — APPROVED** [review]

File: capa/v29.1.0/README.md (hunk 2 above)
Comment (weseven) [reply capa/v29.1.0/README.md]: Absolutely, removed the os-tooling line with the latest commit :+1:

File: capa/v29.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+### Components
+
+- Flatcar from v3815.2.5 to [v3975.2.0](https://www.flatcar.org/releases#release-3975.2.0)
+- Kubernetes from v1.29.7 to [v1.29.8](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.29.md#changelog-since-v1297)
+
+### Apps
+
+- cert-exporter from v2.9.1 to v2.9.2
+- node-exporter from v1.19.0 to v1.20.0
+- observability-bundle from v1.5.2 to v1.6.1
```
Comment (TheoBrigitte) [inline capa/v29.1.0/README.md:14]: :heart:

**Review by TheoBrigitte — APPROVED** [review]

**Pull request merged**